website/docs/cloud-docs/users-teams-organizations/api-tokens.mdx

```diff
@@ -49,6 +49,16 @@ Organization API tokens have permissions across the entire organization. They ca
 
 -> **Note:** This endpoint cannot be accessed with [organization tokens](#organization-api-tokens). You must access it with a [user token](/terraform/cloud-docs/users-teams-organizations/users#api-tokens) or [team token](#team-api-tokens).
 
+## Audit Trails Tokens
+
+Tokens may be generated for reading organization audit trails. They are intended to be used in the [HCP Terraform for Splunk](/terraform/cloud-docs/integrations/splunk) app.
```

Comment:
```suggestion
Tokens may be generated for reading organization audit trails, for example by the [HCP Terraform for Splunk](/terraform/cloud-docs/integrations/splunk) app.
```

Reply:
Style suggestion for active voice and expanding a bit to make this a bit more generic for other metrics integrations:
```suggestion
You can generate an audit trails token to read an organization's [audit trails](/terraform/cloud-docs/api-docs/audit-trails). Use this token type to authenticate integrations pulling audit trail data, for example, using the [HCP Terraform for Splunk](/terraform/cloud-docs/integrations/splunk) app.
```